herzog/parser.py

```diff
@@ -26,8 +26,10 @@ def __init__(self, cell_type: CellType, lines: Iterable[str]):
                 else:
                     self.lines.append(line)
             elif CellType.markdown == self.cell_type:
-                if '"""' == line:
+                if '"""' == line or "pass" == line:
                     pass
+                elif line.startswith('#'):
+                    self.lines.append(line[1:].strip())
```

Comment:
This change to Herzog syntax should be moved into a separate PR. It also should be motivated by explaining the purpose of the change.

Reply:
Opened in #27.